test/functional/layout/test_in_cache.py

```diff
@@ -18,7 +18,7 @@ def conanfile():
                      with_require("base/1.0"))
 
     conan_file += """
-    no_copy_sources = True
+    no_copy_source = True
```

Comment:
This was a typo in the test conanfile, but the behaviour is identical in these tests